### PR TITLE
Merge 3.18.0 into Master

### PR DESCRIPTION
**Pull request by kidster**
**Quality audit reminder**

Doc team: While logged in using your O365 account, complete the quality checklist at http://bit.ly/2hwvBKX. Use the checklist for significant doc changes or additions submitted by you or a non-doc staff contributor.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-43cad472fa/docs/cloud-identity/v2/). :bow:

File: api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+
+   PUT /v2.0/RAX-AUTH/federation/identity-providers/{identityProviderId}
+
+Update an Identity provider (IDP).
+
+.. note::
+
+   - User-admin or User-manage roles can make a request only when the caller's domain is the same as the specified Identity Provider's approvedDomainId.
+   - User-admin or User-manage roles can update the name, description, and emailDomains. Any specified values for other fields are ignored.
+   - A user with the role rcn:admin can make a request only when the caller's domain is within the same RCN as the IDP's specified approvedDomainId.
```
Comment (catlook) [inline api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst]: enclose rcn:admin in double ticks

File: api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+   - A user with the role rcn:admin can update the name, description, emailDomains, and approvedDomainId. Any specified values for other fields are ignored.
```
Comment (catlook) [inline api-docs/api-reference/methods/identity-provider/put-update-identity-provider-v2.0.rst]: enclose rcn:admin in double ticks

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-038756cdbb/docs/cloud-identity/v2/). :bow:

**Pull request merged**